app/scripts/lib/setupSentry.js

```diff
@@ -64,36 +129,141 @@ export const SENTRY_BACKGROUND_STATE = {
     firstTimeFlowType: true,
     seedPhraseBackedUp: true,
   },
   PreferencesController: {
-    currentLocale: true,
     featureFlags: true,
-    forgottenPassword: true,
+    // double-check if contains PID?
     ipfsGateway: true,
-    preferences: true,
+    identities: false,
+    lostIdentities: false,
+    selectedAddress: false,
+    useTokenDetection: true,
+    useNftDetection: true,
+    openSeaEnabled: true,
+    isMultiAccountBalancesEnabled: true,
+    disabledRpcMethodPreferences: true,
+    showTestNetworks: true,
+    isIpfsGatewayEnabled: true,
     useBlockie: true,
     useNonceField: true,
     usePhishDetect: true,
+    dismissSeedBackUpReminder: true,
+    useMultiAccountBalanceChecker: true,
+    use4ByteResolution: true,
+    useCurrencyRateCheck: true,
+    advancedGasFee: true,
+    incomingTransactionsPreferences: true,
+    // double-check if contains PID?
+    knownMethodData: true,
+    currentLocale: true,
+    forgottenPassword: true,
+    preferences: {
+      autoLockTimeLimit: true,
+      showFiatInTestnets: true,
+      showTestNetworks: true,
+      useNativeCurrencyAsPrimaryCurrency: true,
+      hideZeroBalanceTokens: true,
+    },
+    useAddressBarEnsResolution: true,
+    infuraBlocked: true,
+    ledgerTransportType: true,
+    snapRegistryList: true,
+    transactionSecurityCheckEnabled: true,
+    theme: true,
+    isLineaMainnetReleased: true,
+
+    securityAlertsEnabled: false,
+    snapsAddSnapAccountModalDismissed: true,
   },
   SignatureController: {
+    unapprovedMsgs: false,
+    unapprovedPersonalMsgs: false,
+    unapprovedTypedMessages: false,
     unapprovedMsgCount: true,
     unapprovedPersonalMsgCount: true,
     unapprovedTypedMessagesCount: true,
   },
+  SwapsController: {
+    swapsState: {
+      // double-check if contains PID?
+      quotes: false,
+      quotesPollingLimitEnabled: true,
+      fetchParams: true,
+      // double-check if contains PID?
+      tokens: true,
+      tradeTxId: false,
+      approveTxId: false,
+      quotesLastFetched: true,
+      customMaxGas: true,
+      customGasPrice: true,
+      customMaxFeePerGas: true,
+      customMaxPriorityFeePerGas: true,
+      swapsUserFeeLevel: true,
+      selectedAggId: true,
+      customApproveTxData: true,
+      errorKey: true,
+      topAggId: false,
+      routeState: true,
+      swapsFeatureIsLive: true,
+      saveFetchedQuotes: true,
+      swapsQuoteRefreshTime: true,
+      swapsQuotePrefetchingRefreshTime: true,
+      swapsStxBatchStatusRefreshTime: true,
+      swapsStxGetTransactionsRefreshTime: true,
+      swapsStxMaxFeeMultiplier: true,
+      swapsFeatureFlags: true,
+    },
+  },
+  SnapController: {
+    // double-check if contains PID?
+    snaps: true,
+    snapStates: true,
+    snapErrors: true,
+  },
+  CronjobController: {
+    jobs: true,
+  },
+  DesktopController: {
+    desktopEnabled: true,
+  },
+  // we're not sending any data to Sentry from the following controllers
```

Comment:
It should be safe to set all controllers to `{}`, as that will just show us what the top-level controller properties are

Reply:
Resolved in https://github.com/MetaMask/metamask-extension/pull/20595/commits/db9e2566e13811c8ddb6c590cc4370f8e74163f9